src/hooks/useWearables.ts

```diff
@@ -0,0 +1,114 @@
+import { useQuery } from 'react-query';
+import {
+  SyncTypeSettings,
+  ToggleWearableResult,
+  WearableIntegrationFailureCode,
+  WearableIntegrationStatus,
+  WearablesSyncState,
+} from '../components/Wearables/WearableTypes';
+import { useActiveAccount } from './useActiveAccount';
+import { useHttpClient } from './useHttpClient';
+
+/**
+ *
+ * @returns UTC offset in ±HHMM format, e.g., -04:00 for New York City during daylight saving time
+ * or -05:00 during standard time. Equivalent to moment's utcOffset() function.
+ */
+const getUTCOffset = () => {
+  const offsetInMinutes = new Date().getTimezoneOffset();
+  const sign = offsetInMinutes > 0 ? '-' : '+';
+  const hours = Math.abs(Math.floor(offsetInMinutes / 60))
+    .toString()
+    .padStart(2, '0');
+  const minutes = Math.abs(offsetInMinutes % 60)
+    .toString()
+    .padStart(2, '0');
+  const offset = `${sign}${hours}${minutes}`;
+  return offset;
+};
+
+export const useWearableIntegrations = (appId: string) => {
+  const { httpClient } = useHttpClient();
+  const { accountHeaders } = useActiveAccount();
+  return useQuery('/v1/wearables', () =>
+    httpClient
+      .get<WearablesSyncState>('/v1/wearables', {
+        params: { appId },
+        headers: accountHeaders,
+      })
+      .then((res) => res.data),
+  );
+};
+
+export const useWearableIntegration = (ehrId: string) => {
+  const { httpClient } = useHttpClient();
+  const { accountHeaders } = useActiveAccount();
+  return useQuery('/v1/wearables/{ehrId}', () => {
+    httpClient.get(`/v1/wearables/${ehrId}`, { headers: accountHeaders });
+  });
+};
```

Comment:
Cool - I've been thinking about how to go about multiple `useQuery`s grouped together.  There are a lot of repeated httpClient & accountHeaders LOC.  In the FHIR client first pass, I'm trying out having the file expose one hook that wraps several `useQuery`s as callbacks.  That might be the wrong approach and this is the right one - just wanted to mention as we're going to probably hit this in several places.

Reply:
Just to close the loop: as mentioned [here](https://github.com/lifeomic/react-native-sdk/pull/121#discussion_r1160836483), this idea of mine _is_ a bad idea (to have useQuery underneath a useCallback - that'll hit [this issue](https://react.dev/warnings/invalid-hook-call-warning)).  Looks like you're doing it in a [subsequent PR](https://github.com/lifeomic/react-native-sdk/pull/120/files#diff-881ac6549e4fd0710b6837d26da387f05de27be84a226355c28832c3dcab9573R103) just like I did it in [useFhirClient first pass](https://github.com/lifeomic/react-native-sdk/pull/121).  Seems like next iteration is to try out this [lazy query pattern](https://tanstack.com/query/v4/docs/react/guides/disabling-queries#lazy-queries).